### PR TITLE
Schema.org on blog posts

### PR DESCRIPTION
**Pull request by jimmynotjim**
Adding a working example of how to use Schema.org on blog posts for reference in the front-end markup/template playbook.
 
## Additions
 
None
 
## Removals
 
None
 
## Changes
 
- Updated article.html and tags.html to include article components
- Updated footer.html to include organization compontents
 
## Testing
 
Install the [Semantic Inspector plugin](https://chrome.google.com/webstore/detail/semantic-inspector/jobakbebljifplmcapcooffdbdmfdbjh) and navigate locally to:

- /blog/spring-2014-rulemaking-agenda/
 
## Review
 
- @sebworks 
- @anselmbradford 
- @KimberlyMunoz
- and all other FEWDs
 
## Preview
 
screen shot 2015-03-16 at 1 37 44 pm

## Notes
 
- This PR doesn't include full coverage of using Schema.org, but provides a good example of where to start

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -170,24 +170,30 @@ <h2 class="h6">
...
-                        <span class="footer_address-word-break">
-                            Consumer Financial
+                        <span itemprop="name">
+                            <span class="footer_address-word-break">
+                                Consumer Financial
+                            </span>
+                            Protection Bureau<br>
+                        </span>
+                        <span itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
+                            <span itemprop="streetAddress">1700 G Street, NW<br></span>
```
Comment (anselmbradford) [inline _includes/footer.html]: Seems like the schema designated content should only contain content and not markup. So this instead:

```html
<span itemprop="streetAddress">1700 G Street, NW</span><br>
```

Comment (jimmynotjim) [inline _includes/footer.html]: Whoops, while it can contain markup (see the nested span in the name above) it shouldn't contain line breaks. Good catch.

Comment (sebworks) [inline _includes/footer.html]: @jimmynotjim, should I go ahead and update the macros with the schema properties?

Comment (jimmynotjim) [inline _includes/footer.html]: @sebworks I wouldn't just yet, beta clean up is more important. I thought we were supposed to be done w/ the standards stuff yesterday and needed some real examples of Schema in use so I quickly put this together before the meeting. We could even leave this PR open and use it for updating everything with Schema next sprint.

**Comment by anselmbradford** [comment]
schema testing tool https://developers.google.com/structured-data/testing-tool/

**Comment by jimmynotjim** [comment]
closing and will re-open

**Comment by jimmynotjim** [comment]
This should be updated and reopened after launch. It's still useful

**Pull request closed without merging**